### PR TITLE
corrcoef instance method for pytorch frontend and test function

### LINKED ISSUE #26843: corrcoef
corrcoef instance method to pytorch frontend

### PR DESCRIPTION
**Pull request by sharjeelnawaz8182**
# PR Description
corrcoef instance method and test function for pytorch frontend

## Related Issue

Close #26843

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 1]
```diff
@@ -13612,21 +13667,6 @@ def test_torch_tensor_xlogy(
     )
 
 
-# xlogy_
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:13615]: please revert this, it was somehow misplaced above

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 2]
```diff
@@ -5574,6 +5574,61 @@ def test_torch_tensor_copysign_(
     )
 
 
+# xlogy_
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:5577]: this part up till right before corrcoef needs to be restored below

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hello! Thanks for the great work! Please remove all changes to the `.idea` and `docs/demos` folders, thanks!

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py (hunk 1 above)
Comment (sharjeelnawaz8182) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:13615]: which files i should remove ?

**Pull request closed without merging**